hastexo/jobs.py

```diff
@@ -66,15 +66,32 @@ def run(self):
 
         # Get stacks to suspend
         with transaction.atomic():
-            stacks = Stack.objects.select_for_update().filter(
+            stacks_1 = Stack.objects.select_for_update().filter(
```

Comment:
To make this easier to understand and more readable, I'd suggest to call this variable `stacks_without_suspend_by`.

Reply:
Ack.